docs/resources/pingaccess_load_balancing_strategy.md

```diff
@@ -0,0 +1,35 @@
+Create a Load Balancing Strategy.
+
+## Example Usage
+```terraform
+resource "pingaccess_load_balancing_strategy" "demo_strategy" {
```

Comment:
Please can you put the example resource in `pingaccess/test_cases/` as all these files are formatted with `terraform fmt` and will be part of a demonstration test kit.

Reply:
Added a new file under `pingaccess/test_cases -> load_balancing_strategy.tf`